src/main/java/io/cdap/e2e/pages/locators/CdfHubLocators.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * CDF hub related locators.
+ */
+public class CdfHubLocators {
+
+  @FindBy(how = How.XPATH, using = "//input[@class='search-input form-control'][@placeholder='Search by name']")
+  public static WebElement searchTab;
+  @FindBy(how = How.XPATH, using = "//input[@class='search-input form-control'][@placeholder='Search']")
+  public static WebElement searchTabControlCenter;
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='hub-close-btn']")
+  public static WebElement closeButton;
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='one_step_deploy_plugin-btn']")
+  public static WebElement deployButton;
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='wizard-finish-btn']")
+  public static WebElement finishButton;
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'No entities found')]")
+  public static WebElement errorMessage;
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Hub')]")
+  public static WebElement locateHub;
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'Go to homepage')]")
+  public static WebElement toHomePage;
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='Delete']")
+  public static WebElement deleteButton;
+  @FindBy(how = How.XPATH, using = "//button[@id='navbar-hub']")
+  public static WebElement hubPage;
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'No entities found in namespace \"default\"')]")
+  public static WebElement verifyDelete;
```

Comment:
I think verifyDelete can be changed to something else specific to deletion of element like that?